layouts/about-release-schedule.hbs

```diff
@@ -19,6 +19,13 @@
 
         {{{ release-schedule }}}
 
+        <p>
+          Major Node versions enter "Current" release status for six months, which gives library authors time to add support for them.
+          After six months, releases move to "Active" status and are ready for general use.
```

Comment:
This is not exactly true. After six months the release will either be unsupported (odd numbers) or "Active LTS" (even numbers).

Reply:
Amended this in in 8aad80f5, thanks!